share/completions/dmidecode.fish

```diff
@@ -0,0 +1,22 @@
+function __fish_dmidecode_complete_keywords
+	dmidecode -s 2>&1 1>/dev/null | tail -n +4 | string trim
```

Comment:
I'm always a bit wary of magic line number counts.

I think I would turn this into

```fish
dmidecode -s 2>&1 >/dev/null | string match -rg '^\s+(.*)'
```

instead - match lines starting with whitespace instead of skipping exactly 4 lines.

Reply:
Good idea, done.